extensions/amp-story-interactive/0.1/test/validator-amp-story-interactive-results.out

```diff
@@ -77,6 +85,15 @@ amp-story-interactive/0.1/test/validator-amp-story-interactive-results.html:65:8
 |            option-3-text="Uruguay" option-3-results-category="Bunny"
 |            option-4-text="Spain" option-4-results-category="Mouse">
 |          </amp-story-interactive-results>
+|          <amp-story-interactive-results
```

Comment:
Is this meant to be invalid? There's no associated comment for it.

Reply:
Good catch, I'll add a comment for it. It's meant to be invalid since the threshold is not a number